### ISSUE TITLE
WebApplicationFactory not working with Isolated worker model

### ISSUE THREAD

**Issue by doorman02** [Azure/azure-functions-dotnet-worker#2424]
### Description

I am trying to use WebApplicationFactory for integration testing Azure Functions in .Net8 with isolation worker model but getting the following error when starting the test when reaching await host.RunAsync(); in program.cs

```
System.InvalidOperationException: The gRPC channel URI 'http://:' could not be parsed.
   at Microsoft.Extensions.DependencyInjection.GrpcServiceCollectionExtensions.GetFunctionsHostGrpcUri(IConfiguration configuration) in D:\a\_work\1\s\src\DotNetWorker.Grpc\GrpcServiceCollectionExtensions.cs:line 97
   at Microsoft.Extensions.DependencyInjection
```

However it runs fine when launching the Azure functions app and only getting the exception when hosted by the WebApplicationFactory.

Maybe the problem is when I trigger a single test within Rider it will start the process as in-proc instead of isolated worker model?

Any ideas how to configure WebApplicationFactory to use it in isolation worker model?

### Steps to reproduce

Setup .Net 8 Azure Functions App in isolated worker model. Create a test project and use WebApplicationFactory to trigger the endpoint in the Azure Functions App.

Basically what is being done here except in isolated worker model:
https://learn.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-8.0

**Comment by jviau** [comment]
The AspNetCore integration test package is not supported for Azure Functions. The function app is designed to only be started by the function host: https://github.com/azure/azure-functions-host. This host sets up the environment for the worker, without it startup will fail.
